config/environments/staging.rb

```diff
@@ -86,4 +87,4 @@
 end
 
 # Required when using Rails.application.routes.url_helpers from outside the request/response life cycle (models, jobs, lib)
-Rails.application.routes.default_url_options = { host: 'era-test.library.ualberta.ca', port: 443, protocol: 'https' }
+Rails.application.routes.default_url_options = { host: Rails.application.secrets.default_url_host, port: Rails.application.secrets.default_url_port, protocol: 'https' }
```

Comment:
good point. I was capturing what's been overwritten in the Ansible playbook, but I think you are right, no need to have a port that probably will never change into a variable.